src/table/utils/__tests__/handle-key-press.spec.ts

```diff
@@ -553,25 +566,25 @@ describe('handle-key-press', () => {
       evt.shiftKey = true;
       evt.key = 'ArrowUp';
 
-      runBodyHandleKeyPress();
+      runHandleBodyKeyDown();
       expect(evt.preventDefault).toHaveBeenCalledTimes(1);
       expect(evt.stopPropagation).toHaveBeenCalledTimes(1);
-      expect(evt.target.setAttribute).toHaveBeenCalledTimes(1);
+      expect((evt.target as HTMLElement).setAttribute).toHaveBeenCalledTimes(1);
       expect(setFocusedCellCoord).toHaveBeenCalledTimes(1);
       expect(selectionDispatch).toHaveBeenCalledTimes(1);
       expect(handleAccessibility.announceSelectionState).not.toHaveBeenCalled();
       expect(handleScroll.handleNavigateTop).toHaveBeenCalledTimes(1);
     });
 
     it('when press shift + arrow up key on the second row cell, should prevent default behavior, remove current focus and set focus and attribute to the next cell, but not select values for dimension', () => {
-      cell.prevQElemNumber = undefined;
+      cell.prevQElemNumber;
```

Comment:
it does not look correct

Reply:
ah yes, suppose to add a delete before it

Reply:
actually the row is not needed at all. There is no `prevQElemNumber` on the cell by default

Reply:
removed